README.md

```diff
@@ -7,7 +7,7 @@ Gitter via [![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https:
 
 
 ### Goals
-The goals of this project are to extend the MTGJSONv4 protocols and give an option for pre-processed SQLite downloads.
+The goals of this project are to extend the MTGJSONv5 protocols and give an option for pre-processed SQLite downloads.
 lly edit it to be correct. Once that is accomplished, we are then no longer dependent on them for card data, except for rullings.
```

Comment:
`lly edit it to be correct.`

Typo? What should it mean?

Reply:
This is fine for now. If you want something more complex like a link to the other repo please do so in another MR.

Reply:
Your reaction is probably related to the PR description, not my comment above which you replied to but did not answer. @staghouse 

I would like to see the typo fixed, but I can not tell what it should mean.

> (fom the [README](https://github.com/mtgjson/mtgsqlive/blob/master/README.md#goals))
>
>**Goals**
>
>The goals of this project are to extend the MTGJSONv5 protocols and give an option for pre-processed SQLite downloads. lly edit it to be correct. Once that is accomplished, we are then no longer dependent on them for card data, except for rullings.


The second sentence is messed up: `lly edit it to be correct.`

Reply:
I think actually we can remove that entire section, thoughts? CC @ZeldaZach @omfgitsmark